lib/apex/format.ex

```diff
@@ -116,13 +116,35 @@ defimpl Apex.Format, for: Map do
   end
 
   def format(data, options) do
+    data = convert_keys_to_atoms(data, options)
     Apex.Format.Seq.format(
       Map.to_list(data),
       options,
       start_token: "\%{",
       end_token: "}",
       numbers: false) |> colorize(data, options)
   end
+
+  defp keys_to_atoms?(options) do
+    options[:keys_to_atoms] == true
+  end
+
+  defp convert_keys_to_atoms(data, options) do
+    data =
```

Comment:
The change above should simplify the body of the function to

```elixir
def keys_to_atoms(data, true) do
   Map.new(data, fn {k,v} -> {convert_to_atom(k), v} end)
end
```
Please also take into account that the code should also be able to handle non atom and non string keys. Currently the code would try to invoke `String.to_atom` also on other types and would blow up.

```elixir
def convert_to_atom(key)
def convert_to_atom(key) when is_binary(key), do: String.to_atom(key)
def convert_to_atom(key), do: key
```